### DISCUSSION TITLE
JKbms

### DISCUSSION THREAD

**Discussion by donaldbrian** [jblance/mpp-solar#248] [General]
I have a JK-BD6A20S10P.
GetCellData returns pretty much useless info.

~$ jkbms -p C8:47:8C:E2:8D:26 -c getCellData
Command: getCellData - BLE Cell Data inquiry
------------------------------------------------------------
Parameter                       Value           Unit
header                          55aaeb90           
record_type                     02                 
record_counter                  213                 
voltage_cell01                  3.806417863259254e-31   V   
...

voltage_cell19                  2.1019476964872256e-43  V   
...
resistance_cell01               0.0             Ohm 
...
resistance_cell24               2.9626331329345703      Ohm 
...
average_cell_voltage            1.7516230804060213e-42  V   
delta_cell_voltage              2.350988701644575e-38   V   
highest_cell                    1                   
lowest_cell                     1                   
flags                           0000               
uptime                          0D0H0M0S           
checksum                        dc                 
highest_cell_voltage            3.806417863259254e-31   V   
lowest_cell_voltage             3.806417863259254e-31   V

```

I have **8** cells.
1.7516230804060213e-42 V?

Anything I could try?

**Comment by jblance** [comment]
try adding `-P JK02` to the command (to use a different protocol)

otherwise add debug to both commands `-D` and post the info

**Comment by donaldbrian** [comment]
'-P JK02' works. Thanks.
Funny because I **did** try that (got it from the up-arrow command history) and it returned some error.
